
recorder:
  joint_states_topic: "/joint_states"
  joints: # joints to observe
        - iiwa_joint_1
        - iiwa_joint_2
        - iiwa_joint_3
        - iiwa_joint_4
        - iiwa_joint_5
        - iiwa_joint_6
        - iiwa_joint_7
  record_rate: 20     # in Hertz

player:
  action_server: /CartesianImpedance_trajectory_controller/follow_joint_trajectory